src/makers/hummingbird/hummingbird.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Your icon is not rendering because there is no fill color specified. Ideally you would use `currentColor` on all elements. Something like...

```xml
<?xml version="1.0" encoding="iso-8859-1"?>
<!-- Generator: Adobe Illustrator 22.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="&#x56FE;&#x5C42;_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 96 96" style="enable-background:new 0 0 96 96;" xml:space="preserve">
	<g>
		<path d="M88,8v80H8V8H88 M88,6H8C6.9,6,6,6.9,6,8v80c0,1.1,0.9,2,2,2h80c1.1,0,2-0.9,2-2V8C90,6.9,89.1,6,88,6L88,6z" fill="currentColor" />
	</g>
	<g>
		<g>
			<polygon points="28.1,46.8 26.8,46.8 26.8,45 25.6,45 25.6,46.8 25.6,48.1 25.6,49.9 26.8,49.9 26.8,48.1 28.1,48.1 28.1,49.9
			29.3,49.9 29.3,48.1 29.3,46.8 29.3,45 28.1,45 		" fill="currentColor" />
			<polygon points="33.1,48.7 31.8,48.7 31.8,46.8 30.6,46.8 30.6,49.9 31.4,49.9 31.8,49.9 33.1,49.9 33.8,49.9 34.3,49.9
			34.3,46.8 33.1,46.8 		" fill="currentColor" />
			<path d="M59.3,46.8H58v1.2v0.6v1.2h1.2h1.2v0.6H58v1.2h3.8v-0.3v-0.9v-0.6v-1.2v-0.6v-1.2h-1.2H59.3z M60.5,48.7h-1.2v-0.6h1.2
			V48.7z" fill="currentColor" />
			<polygon fill="currentColor" points="54.3,46.8 53,46.8 53,48.1 53,49.9 54.3,49.9 54.3,48.1 55.5,48.1 55.5,49.9 56.8,49.9 56.8,48.1 56.8,46.8
			55.5,46.8 		" />
			<path fill="currentColor" d="M64.3,46.8V45H63v4.9h1.2h1.2h1.2v-1.2v-0.6v-1.2h-1.2H64.3L64.3,46.8z M65.5,48.7h-1.2v-0.6h1.2V48.7z" />
			<polygon fill="currentColor" points="39.3,46.8 38.1,46.8 36.8,46.8 36.1,46.8 35.6,46.8 35.6,49.9 36.8,49.9 36.8,48.1 38.1,48.1 38.1,49.9
			39.3,49.9 39.3,48.1 40.6,48.1 40.6,49.9 41.8,49.9 41.8,48.1 41.8,46.8 40.6,46.8 		" />
			<polygon fill="currentColor" points="46.8,46.8 45.5,46.8 44.3,46.8 43.7,46.8 43,46.8 43,49.9 44.3,49.9 44.3,48.1 45.5,48.1 45.5,49.9 46.8,49.9
			46.8,48.1 48,48.1 48,49.9 49.3,49.9 49.3,48.1 49.3,46.8 48,46.8 		" />
			<rect x="50.5" y="46.8" width="1.2" height="3.1" fill="currentColor" />
			<rect x="50.5" y="45" width="1.2" height="1.2" fill="currentColor" />
			<rect x="68" y="46.8" width="1.2" height="3.1" fill="currentColor" />
			<polygon points="70.5,46.8 70.5,48.1 70.5,49.9 71.8,49.9 71.8,48.1 73,48.1 73,46.8 		" fill="currentColor" />
			<rect x="68" y="45" width="1.2" height="1.2" fill="currentColor" />
			<path d="M76.8,45v1.8h-1.2h-1.2l0,0v3.1l0,0h1.2h1.2H78V45C78,45,76.8,45,76.8,45z M76.8,48.7h-1.2v-0.6h1.2V48.7z" fill="currentColor" />
		</g>
		<path d="M21.5,45.5l0.2-0.8l2,0.5v-1h-5.6l3.1,1.5l0.6,2.5l-3.8-4H18v4.7l2.6-1.7l1,1L18,49.7v0.2h5.7v-4.4L21.5,45.5z M22,48.2
		l-0.5-2.5h0.8L22,48.2z" fill="currentColor" />
	</g>
</svg>
```